fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/common/JdbcTarget.java

```diff
@@ -33,6 +36,10 @@ public JdbcTarget(Connection c) {
 
     @Override
     public void runStatement(IDatabaseTranslator translator, String ddl) {
+        if (logger.isLoggable(Level.FINEST)) {
```

Comment:
Why finest?  typically we target FINE? this is a curiosity question.

Reply:
It generates a lot of output and we already log a ton under FINE so I was trying to avoid making it any worse.